src/seedu/addressbook/data/person/PostalCode.java

```diff
@@ -0,0 +1,26 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents the postal code in a Person's address in the address book.
+ * Guarantees: immutable
+ */
+public class PostalCode {
+    
+    private final String value;
```

Comment:
while there is nothing too bad calling the postal code data as value, it may be clearer if you used a better variable name.